README.md

```diff
@@ -1,48 +1,120 @@
-# README
+* `# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Fortytude is an object oriented based website for the Temple University Libraries.
 
-Things you may want to cover:
+## System Requirements
 
-* Ruby version
+- Ruby 2.5.1
+- Google OmniAuth2
 
-* System dependencies
+##  Getting Started
 
-Install Ruby Gem dependencies
+* Set up environment variables:
 
-`bundle install`
+```
+export GOOGLE_OAUTH_CLIENT_ID="Google client ID goes here"
+export GOOGLE_OAUTH_SECRET="Google OAuth secret goes here"
+```
+Add these same lines to your `.bash_profile` or `.bashrc` file, depending on
+how you've setup your shee, depending on how you've setup your Bash shell:
 
-* Configuration
+* Clone the repository and navigate to the souce code directory
 
-* Database creation
+```
+git clone git@github.com:tulibraries/fortytude.git`
+`cd fortytude
+```
 
-Create data tables
+* Install dependencies
 
-`rake db:migrate`
+```
+bundle install
+```
 
-Seed initial user from the command line
+* Create database tables
 
-`rails runner 'Account.create(email: "<YOURTUACCESSID>@temple.edu", admin: true, password: Devise.friendly_token[0,20]).save'`
+```
+bundle exec rake db:migrate
+```
 
-*Or* Create a account seed file 
+* Populate the database with fake data
 
-`mv db/account_seeds.rb.example db/account_seeds.rb`
+```
+bundle exec rake db:populate
+```
+
+* Seed initial user from the command line
```

Comment:
THis would be a goo place to be more explicit about the format the emails need to be in

Reply:
✔️